bin/publish

```diff
@@ -96,13 +96,13 @@ function update-exocom-package-versions
 function update-exocom-bus-number
   try
     replace.sync do
-      files: ['features/**/*.feature', 'exosphere-shared/**/*.yml']
-      replace: /bus:(\s*)type: exocom(\s*)version: \d+.\d+.\d+/g
+      files: ['**/*.feature', 'exosphere-shared/**/*.yml']
+      replace: /bus:\n(\s*)type: exocom\n(\s*)version: \d+.\d+.\d+/g
       with:
         """
         bus:
-          type: exocom
-          version: #{@target-version}
+        $1type: exocom
```

Comment:
`$1` is the captured whitespace in front of type. I ran this function locally and it appeared correct